movie_REST_API/movie_REST_API/Service/NetworkService.swift

```diff
@@ -1,66 +1,65 @@
-// ApiService.swift
+// NetworkService.swift
 // Copyright © RoadMap. All rights reserved.
 
 import Foundation
 
-/// Get Data - сетевой слой
-final class ApiService {
-    
-    // MARK: - Private Constant
-    private enum Constant {
+/// Протокол сетевого слоя
+protocol NetworkServiceProtocol {
+    func getMoviesData(moviesURL: String, completion: @escaping (Result<MoviesData?, Error>) -> ())
+    func getActorData(actorURL: String, completion: @escaping (Result<ActorData?, Error>) -> ())
+}
+
```

Comment:
```suggestion
/// Протокол сетевого слоя
protocol NetworkServiceProtocol {
    func getMoviesData(moviesURL: String, completion: @escaping (Result<MoviesData?, Error>) -> ())
    func getActorData(actorURL: String, completion: @escaping (Result<ActorData?, Error>) -> ())
}

```
протокол в отдельный файл